rules/opentitan.bzl

```diff
@@ -204,7 +204,7 @@ elf_to_scrambled_rom_vmem = rv_rule(
     },
 )
 
-def _bin_to_flash_vmem_impl(ctx):
+def _bin_to_vmem_impl(ctx):
```

Comment:
I think @timothytrippel was making a useful distinction here, that the vmem for flash files requires a different operation than the vmem for the ROM.